doc/transform_methods/linbasex.rst

```diff
@@ -0,0 +1,156 @@
+.. |nbsp| unicode:: 0xA0 
+   :trim:
+
+Lin-Basex
+=========
+
+
+Introduction
+------------
+
+Inversion procedure based on 1-dimensional projections of VM-images as 
+described in Gerber et al. [1]. 
+
+[ *from the abstract* ]
+
+*VM-images are composed of projected Newton spheres with a common centre. 
```

Comment:
I don't mind either way, but if we want to keep all the docs in `Merrican English, then "centre" should be "center".